<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>697</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QPushButton" name="F1">
      <property name="text">
       <string>F1</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="F1_dir">
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="F2">
      <property name="text">
       <string>F2</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="F2_dir">
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="F3">
      <property name="text">
       <string>F3</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="F3_dir">
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="F4">
      <property name="text">
       <string>F4</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLabel" name="F4_dir">
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QPushButton" name="F5">
      <property name="text">
       <string>F5</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLabel" name="F5_dir">
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QPushButton" name="F6">
      <property name="text">
       <string>F6</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLabel" name="F6_dir">
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QPushButton" name="F7">
      <property name="text">
       <string>F7</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLabel" name="F7_dir">
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QPushButton" name="F8">
      <property name="text">
       <string>F8</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QLabel" name="F8_dir">
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QPushButton" name="F9">
      <property name="text">
       <string>F9</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QLabel" name="F9_dir">
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QPushButton" name="F10">
      <property name="text">
       <string>F10</string>
      </property>
     </widget>
    </item>
    <item row="9" column="1">
     <widget class="QLabel" name="F10_dir">
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </item>
    <item row="10" column="0">
     <widget class="QPushButton" name="F11">
      <property name="text">
       <string>F11</string>
      </property>
     </widget>
    </item>
    <item row="10" column="1">
     <widget class="QLabel" name="F11_dir">
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </item>
    <item row="11" column="0">
     <widget class="QPushButton" name="F12">
      <property name="text">
       <string>F12</string>
      </property>
     </widget>
    </item>
    <item row="11" column="1">
     <widget class="QLabel" name="F12_dir">
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </item>
    <item row="13" column="0" colspan="2">
     <widget class="QPushButton" name="getFile">
      <property name="text">
       <string>Выбрать файл с настройками</string>
      </property>
     </widget>
    </item>
    <item row="12" column="0" colspan="2">
     <widget class="QPushButton" name="saveButton">
      <property name="text">
       <string>Сохранить текущие настройки в файл</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>697</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
